js/src/behaviors/popover.ts

```diff
@@ -123,13 +177,23 @@ function positionCaret(
 	desiredLeft: number
 ) {
 	const contentWidth = popoverContent.offsetWidth;
-	const buttonCenter = summaryButton.offsetLeft + summaryButton.offsetWidth / 2;
+	const buttonWidth = summaryButton.offsetWidth;
+	const buttonLeft = summaryButton.offsetLeft;
+	const popover = popoverContent.closest('.popover') as HTMLElement;
+	const isRtl = isRTL(popover);
 
-	const buttonCenterRelativeToContent = buttonCenter - desiredLeft - 4;
-	const caretLeftPercent = (buttonCenterRelativeToContent / contentWidth) * 100;
+	const buttonCenter = buttonLeft + buttonWidth / 2;
+	let caretPosition;
 
-	const clampedCaretLeftPercent = Math.min(Math.max(caretLeftPercent, 10), 90);
-	popoverContent.style.setProperty('--caret-left', `${clampedCaretLeftPercent}%`);
+	if (!isRtl) {
+		caretPosition = ((buttonCenter - desiredLeft - 4) / contentWidth) * 100;
+	} else {
+		const ltrPosition = ((buttonCenter - desiredLeft + 4) / contentWidth) * 100;
+		caretPosition = 100 - ltrPosition;
+	}
```

Comment:
Stylistic nit: can we update this so the check is `if(isRtl)`? That way, there's less negation to follow.